integrationExamples/gpt/airgridRtdProvider_example.html

```diff
@@ -59,6 +59,7 @@
                   apiKey: "key123",
                   accountId: "sdk",
                   publisherId: "pub123",
+                  bidders: ["appnexus", "pubmatic"],
```

Comment:
@ydennisy this is needed so the data will be passed to the bidders.
I'm including this here so the example will be fully functional.
Please, let me know if I shouldn't.

Reply:
read this array out of publisher config